app/src/main/AndroidManifest.xml

```diff
@@ -12,6 +12,7 @@
         android:theme="@style/AppTheme">
         <activity
             android:name=".activities.MainActivity"
+            android:theme="@style/Theme.AppCompat.Light.NoActionBar"
```

Comment:
Why does the activity need its own theme declaration when it uses the same theme as the app? I think you can just remove this line